src/BoltFill.svg

```diff
@@ -0,0 +1,4 @@
+
+<svg width="11" height="17" viewBox="0 0 11 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This icon appears off-center unless I match the [viewbox of the original](https://github.com/artsy/palette-mobile/blob/1ff6e6c9249e3a4848cb6096cf6c76e989418ceb/src/svgs/BoltFill.tsx#L7).

Is that ok for these legacy icons?

Reply:
As long as its ok to use without adjustment it's fine. Happy to fire up Illustrator if needed.

Reply:
Yea, my fear is adjusting the dimensions would have ripple effects when go update the call sites in Eigen/Energy, so being able to swap the imports without any fuss is what I'll prioritize. Just making sure the 18x18 wasn't inviolable…

Reply:
oh nah its fine. we have other icons in different sizes (the logos for instance)